src/AppBundle/Controller/BundleController.php

```diff
@@ -127,8 +140,14 @@ public function showBundlesListAction(Request $request, $page = 1, $order = 'def
 
         // Get content of Bundles List page
         $query = new Query();
-        $criterion = new Query\Criterion\LocationId($this->bundlesListLocationId);
+        $criterion = new Query\Criterion\LogicalAnd(
```

Comment:
Not sure to understand why you are changing the approach here

Reply:
I changed this because I thought that here I must add condition to get related contents with given category.